.github/workflows/cicd.yml

```diff
@@ -58,8 +59,6 @@ jobs:
       SKIP_DEPS: 1
     steps:
       - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
```

Comment:
I think you still need this for cov?

Reply:
I don't think so, at least from the behaviour in this repo. The `codecov` package seems to handle reporting correctly, issue was a misconfiguration on codecov website.